nix/jormungandr.nix

```diff
@@ -34,38 +34,8 @@
 }:
 
 let
-  release = rec {
-    version = "0.8.6";
-    # Git revision of input-output-hk/jormungandr repo.
-    rev = "v${version}";
-    # Hash of git repo and all of its submodules.
-    sha256 = "02xwdxmq32xwjszag8avpcdqakdhhy17mqqnngpn8f9viwbmhvf8";
-    # Hash of all Cargo dependencies.
-    cargoSha256 = "1vjlijccaxibn5i56x74rjzbqfrwhl1aq7l8rp6linnkvh01jx15";
-  };
-
-  windows = rec {
-    # URL and hash of windows binary release
-    url = "https://github.com/input-output-hk/jormungandr/releases/download/v${release.version}/jormungandr-v${release.version}-x86_64-pc-windows-msvc.zip";
-    sha256 = "199hav5rw4ic7lhgx6wj65bkm9ch1bgzxwc2q6p2kj71frx9hwkw";
-  };
-
-  jormungandr-win64 = pkgs.runCommand "jormungandr-win64-${release.version}" {
-    nativeBuildInputs = [ pkgs.buildPackages.unzip ];
-  } ''
-    mkdir -p $out/bin
-    cd $out/bin
-    unzip ${pkgs.fetchurl windows}
-  '';
-
-  nonWindows = pkg: if pkgs.stdenv.hostPlatform.isWindows
-    then jormungandr-win64
-    else pkg;
-
-in rec {
-  jormungandr = nonWindows (iohkLib.rust-packages.pkgs.makeJormungandr release);
-  jormungandr-cli = nonWindows (iohkLib.rust-packages.pkgs.makeJcli release);
-
-  inherit jormungandr-win64;
-  inherit (jormungandr) src;
-}
+  iohk-nix = import (fetchTarball {
+    url = https://github.com/input-output-hk/iohk-nix/archive/e0b995585478f1e2fb417123cc224054092090a1.tar.gz;
+    sha256 = "0igigjk54y8d50wpwbx3p9biqa098af95xjmi50sy6f5sszjvk3s";
+  }) {};
+in { inherit (iohk-nix.jormungandrLib.packages.v0_8_7) jormungandr jcli; }
```

Comment:
Hmmm. This seems wrong. The above was made in order to enable us to use a different version that the one used by iohk-nix. Often, iohk-nix is lagging behind or, on the contrary, sometimes ahead, breaking our own CI. We are managing the whole version of Jörmungandr ourselves on purpose.

What was the reason to remove all this?

Reply:
Reason: circumvent the build failure I was seeing. https://input-output-rnd.slack.com/archives/CAP8NM7N0/p1579861205062100

Alternatively, we could bump the rust version (as Michael suggested, and seems logical).

Not sure where the rust version is specified though 🤔

Reply:
I suppose that's the way to go -> https://github.com/input-output-hk/cardano-wallet/pull/1304/commits/900f3f7477ba6644b718f7c5e4dfbd1d0a381a45

After this and bumping all hashes to jörmungandr 0.8.7 I was able to run successfully on local box:

```
nix-build nix/jormungandr.nix
```